Examples/Detectors/Geant4Detector/src/MockupSectorBuilder.cpp

```diff
@@ -230,40 +217,42 @@ ActsExamples::MockupSectorBuilder::BuildSector(
   auto transform = Acts::Transform3(Acts::Translation3(pos));
   transform *= rotZ;
 
-  // keep the inner, middle and outer chambers in order to constraint them in
-  // cylinder bounds
-  std::vector<std::shared_ptr<Acts::Experimental::DetectorVolume>>
-      detectorVolumesOfSector = {};
-  std::vector<std::shared_ptr<Acts::Experimental::DetectorVolume>>
-      inner_detectorVolumesOfSector = {};
-  std::vector<std::shared_ptr<Acts::Experimental::DetectorVolume>>
-      middle_detectorVolumesOfSector = {};
-  std::vector<std::shared_ptr<Acts::Experimental::DetectorVolume>>
-      outer_detectorVolumesOfSector = {};
-
+  //create a vector for the shifted surfaces of each chamber
   std::vector<std::shared_ptr<Acts::Surface>> shifted_surfaces = {};
 
+  //creare an array of vectors that holds all the chambers of each sector
+
+  const int array_size =  det_volumes.size();  
```

Comment:
You can move this line at l193 and use `array_size` (should be `arraySize`).

Reply:
actually i think we said to do that directly:
  ```
 std::vector<std::unique_ptr<Acts::CylinderVolumeBounds>>
      cylinderVolumesBounds(detVolumes.size());
``` 
?? is it better?